DXRModules/DeusEx/Classes/DXRFixup.uc

```diff
@@ -219,12 +252,20 @@ function PreFirstEntry()
     FixHolograms();
     FixShowers();
 
-#ifdef vanilla
-    foreach AllActors(class'#var(prefix)Lamp', lmp) {
-        lmp.InitLight();
+    if (#defined(vanilla)) {
```

Comment:
this means the code is attempted to be compiled in other mods as `if(false) {` which means it will fail since Lamp::InitLight doesn't exist in other mods